pureStorageDriver/README.md

```diff
@@ -17,7 +19,7 @@ This helm chart installs the Pure Service Orchestrator CSI plugin on a Kubernete
   - Ubuntu 18.04
 - #### Environments Supported*:
   - Kubernetes 1.13+
-    - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico 
+    - NOTE: for Kubernetes 1.17, there is an [issue](https://github.com/kubernetes/kubernetes/issues/87852) using vxlan with Flannel or Calico, the issue has been fixed in 1.18.5 release.
```

Comment:
Can we provide a specific set of ranges of versions that have the issue (if we know it)? Something like "from k8s 1.17.0 to 1.17.8, and from 1.18.0 to 1.18.5", so that there's no ambiguity?

Reply:
good idea, I do need to do more tests to find out the actual range